experiments/graph_oriented_layout_design/gold_cost_objectives.cpp

```diff
@@ -0,0 +1,144 @@
+//
+// Created by Simon Hofmann on 30.08.24.
+//
+#include "fiction/layouts/cartesian_layout.hpp"
+#include "fiction/layouts/clocked_layout.hpp"
+#include "fiction/layouts/gate_level_layout.hpp"
+#include "fiction/layouts/tile_based_layout.hpp"
+#include "fiction/networks/technology_network.hpp"
+#include "fiction/types.hpp"
+#include "fiction_experiments.hpp"
```

Comment:
In the experiments, we treat `fiction` as a library and thus use angular brackets (`<>`) for the includes. 

`fiction_experiments.hpp` on the other hand is a part of the experiments framework, thus quote includes (`""`) are correct here.